ports/nethost/portfile.cmake

```
@@ -1,19 +1,20 @@
-set(COMMIT_HASH v7.0.0)
+set(PRODUCT_VERSION "8.0.0")
+set(COMMIT_HASH "v${PRODUCT_VERSION}")
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO dotnet/runtime
     REF ${COMMIT_HASH}
```

```suggestion
    REF "v${VERSION}"
```
${VERSION} is obtained from vcpkg.json.